lexicons/com/atproto/admin/getModerationReports.json

```diff
@@ -10,6 +10,15 @@
         "properties": {
           "subject": {"type": "string"},
           "resolved": {"type": "boolean"},
+          "actionType": {
+            "type": "string",
+            "knownValues": [
+              "com.atproto.admin.defs#takedown",
+              "com.atproto.admin.defs#flag",
+              "com.atproto.admin.defs#acknowledge",
+              "com.atproto.admin.defs#escalate"
+            ]
+          },
```

Comment:
It feels a bit of a bummer we can't transclude this list by ref. will have to update in both places if we add additional action types.

not that bad in this case though, probably not much more churn on this enum.

Reply:
I agree—I originally wrote it with a ref then realized it doesn't quite work.  I can imagine adding support for that to lexicon, though— the funny part is that you have to resolve the ref in order to know whether the parameter is valid, whereas right now the params can be validated statically.

Reply:
Logged an issue: https://github.com/bluesky-social/atproto/issues/1030